laplace/baselaplace.py

```diff
@@ -442,8 +454,13 @@ def _gridsearch(
                     loss_with_var=loss_with_var,
                     dict_key_y=self.dict_key_y,
                 )
-            except RuntimeError:
+            except LinAlgError:
```

Comment:
@runame please recheck. It seems PyTorch now uses `LinAlgError` for this https://github.com/pytorch/pytorch/pull/68571. But I also added your suggestion for safety.